lighthouse-core/test/audits/byte-efficiency/legacy-javascript-test.js

```diff
@@ -98,28 +98,33 @@ describe('LegacyJavaScript audit', () => {
     ]);
     expect(result.items).toHaveLength(1);
     expect(result.items[0]).toMatchInlineSnapshot(`
+Object {
```

Comment:
indentation issue with these files?

Reply:
oops. You have to babysit jest when asking it to `-u` inline snapshots. It was fixed for a short minute... but regressed at some point. Maybe fixed in latest, idk.